src/components/Navbar.jsx

```diff
@@ -1,12 +1,27 @@
 // LIBRARY IMPORTS
 import React from 'react';
+import { NavLink } from 'react-router-dom';
 
 // LOCAL IMPORTS
 
 export default function NavBar() {
   return (
-    <div>
-      Navigation Bar will go here.
+    <div className="space-x-4">
+      <NavLink
+      exact
+      to="/"
+      className="px-4 py-2 bg-base-100 text-primary rounded-full font-bold hover:bg-red-500 hover:text-white"
+      activeClassName="bg-red-500 text-white border-red-500"
```

Comment:
`activeClassName` is a prop you can use with `NavLink`. When the route matches (i.e., when the link is "active"), NavLink will automatically apply the class provided to `activeClassName`. It's a way of saying, "When this link points to the current route, apply these styles."
This is where I'm having issues! Would love your help here @razananisathar @shyusu4 @Havana82  I had to add the hover style props above to `className` since the props here in `activeClassName` alone were not resulting in the red on hover/selection - just a default grey.
I had initially used DaisyUI's button component (by adding `btn` to the class names) - but that didn't seem to mesh well with React so I'm only using DaisyUI custom theme colors with Tailwind classes

Reply:
Maybe you could try to move the classes from `activeClassName` and add an `active:` state to `className`?